[docs] Replace some images, gifs with videos (#2662)
mui/mui-toolpad: docs/data/toolpad/concepts/theming.md

## Adding a global theme

<video controls width="auto" height="100%" style="contain" alt="custom-component-creation">
  <source src="/static/toolpad/docs/concepts/theming/theme-1.mp4" type="video/mp4">
  Your browser does not support the video tag.
</video>

> 1. In order to use a custom theme, go to the **Theme** tab in the inspector panel and click on **Add theme**:

Reviewer: This button has been removed recently

```suggestion
1. To change the application theme, go to the **Theme** tab in the inspector panel.
```
